container/src/constants/communication.ts

```diff
@@ -1,151 +1,161 @@
 
   /**
-   * A message emitted from the micro frontend to report that the navigation is completed is sent
+   * Event fired when the micro frontend requests to report that the navigation is completed.
    */
   export const NAVIGATION_COMPLETED_REPORT = 'report-navigation-completed-request';
 
   /**
-   * A message emitted from the micro frontend when a request to update the modal path parameters is sent
+   * Event fired when the micro frontend requests to update the modal path parameters.
    */
   export const UPDATE_MODAL_PATH_DATA_REQUEST = 'update-modal-path-data-request';
 
   /**
-   * A message emitted from the micro frontend when a request to check on the validity of a path is sent
+   * Event fired when the micro frontend requests to check on the validity of a path.
    */
   export const CHECK_PATH_EXISTS_REQUEST = 'check-path-exists-request';
 
   /**
-   * A message emitted from the micro frontend when a request to set the 'dirty status' (eg: avoid closing if unsaved changes) is sent
+   * Event fired when the micro frontend requests to set the 'dirty status' (e.g. avoid closing if unsaved changes).
    */
   export const SET_DIRTY_STATUS_REQUEST = 'set-dirty-status-request';
 
   /**
-   * A message emitted from the micro frontend when a request to set the view group data
+   * Event fired when the micro frontend requests to set the view group data.
    */
   export const SET_VIEW_GROUP_DATA_REQUEST = 'set-viewgroup-data-request';
 
   /**
-   * A message emitted from the micro frontend when a request to set the document title
+   * Event fired when the micro frontend requests to set the document title.
    */
   export const SET_DOCUMENT_TITLE_REQUEST = 'set-document-title-request';
 
   /**
-   * A message emitted from the micro frontend when a request to open user settings
+   * Event fired when the micro frontend requests to open the user settings.
    */
   export const OPEN_USER_SETTINGS_REQUEST = 'open-user-settings-request';
 
   /**
-   * A message emitted from the micro frontend when a request to close user settings
+   * Event fired when the micro frontend requests to close the user settings.
    */
   export const CLOSE_USER_SETTINGS_REQUEST = 'close-user-settings-request';
 
   /**
-   * A message emitted from the micro frontend when a request to collapse left side navigation
+   * Event fired when the micro frontend requests to collapse left side navigation.
    */
   export const COLLAPSE_LEFT_NAV_REQUEST = 'collapse-leftnav-request';
 
   /**
-   * A message emitted from the micro frontend when a request to remove the backdrop
+   * Event fired when the micro frontend requests to update the top navigation.
    */
   export const UPDATE_TOP_NAVIGATION_REQUEST = 'update-top-navigation-request';
 
   /**
-   * A message emitted from the micro frontend when a request to remove the backdrop
+   * Event fired when the micro frontend requests to check if the path exists.
```

Comment:
```suggestion
   * Event fired when the micro frontend requests to check if a path exists.
```
Assuming that the path is a parameter of some sort.

Reply:
No, it's not a parameter, it can be the hole path or parts of it like /projects/project/settings or just /project/settings